model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+groups:
+  - id: metric.otel.sdk.span.created_count
```

Comment:
Consider "started_count" (started and ended would better match the mental model from a consumer).

We use "start" here https://github.com/open-telemetry/opentelemetry-proto/blob/ffade295895a2be5a6e7931eb0cda1c72bc4c0f6/opentelemetry/proto/trace/v1/trace.proto#L191.
We use "create" here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span-creation.

Reply:
ref to previous thread on this: https://github.com/open-telemetry/semantic-conventions/pull/1631#discussion_r1924121481

Reply:
@reyang please follow up in the linked thread (and unresolve it) if you don't agree with the reasoning there.